content/learning-paths/cross-platform/simd-on-rust/simd-on-rust-part2.md

```diff
@@ -128,7 +128,7 @@ Shown below is the disassembly output for the `sad_neon` function:
 
 You will notice the use of `uabd` and `udot` assembly instructions that correspond to the `vabdq_u8`/`vdotq_u32` intrinsics.
 
-Now create an equivalent Rust program using `std::arch` `dotprod` intrinsics. Save the contents shown below in a file named `dotprod2.rs`:
+Now create an equivalent Rust program using `std::arch` `neon` intrinsics. Save the contents shown below in a file named `dotprod2.rs`:
```

Comment:
no, this was correct, please revert the change, this part checks for the dotprod extension.

Reply:
neon does not imply dotprod, but dotprod implies neon.